### PR TITLE
Create code gen tool for Prims op | feat(atenlib)

### PR DESCRIPTION
**Pull request by fatcat-z**
1. Implement a code gen tool for all prims op.
2. Upload the first prims.py file which contains all prims op.
3. Make sure the generated prims.py file could pass our lintrunner check.
4. Finish the item 1 in #418.

### CODE REVIEW

File: onnxscript/function_libs/tools/torch_prims/generate_prims_signatures.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+        {dashline}
+        # mypy: disable-error-code="misc,arg-type,type-arg,valid-type,assignment,return-value"
+        """
+    )
+
+
+def _get_func_schema_in_namespace(namespaces: List[_OpNamespace]) -> Dict[str, FunctionSchema]:
+    table: Dict[str, FunctionSchema] = {}
+    for op_namespace in namespaces:
+        for attr_name in dir(op_namespace):
```
Comment (titaiwangms) [inline onnxscript/function_libs/tools/torch_prims/generate_prims_signatures.py:248]: Not sure if torch.ops.prims has missing ops issue like we had torch.ops.aten (https://github.com/pytorch/pytorch/issues/99681). But reverse look up is generally preferred I think. Please take a look at https://github.com/pytorch/pytorch/pull/99682

Comment (fatcat-z) [reply onnxscript/function_libs/tools/torch_prims/generate_prims_signatures.py:248]: So far, there is no such problem for prims ops. But I agree we should find a way to guarantee the op list after all prims ops are stable. In the [doc](https://pytorch.org/docs/master/ir.html#prims-ir), pirms ops are still under development so we don't have a full list for them.

**Review by xiaowuhu — APPROVED** [review]

**Review by justinchuby — CHANGES_REQUESTED** [review]
Avoid rushing PRs on weekends

**Comment by fatcat-z** [comment]
Create another [PR](https://github.com/microsoft/onnxscript/pull/660) which contains only 1 comment change. It has the failures as same as this one, which is caused by test_onnx_func_expe function. So merge this into main branch.

File: onnxscript/function_libs/tools/torch_aten/generate_prims_signatures.py
Comment (justinchuby) [inline onnxscript/function_libs/tools/torch_aten/generate_prims_signatures.py:1]: Is the intended path torch_aten?

**Review by justinchuby — APPROVED** [review]
Amazing work. Thank you!

File: onnxscript/function_libs/tools/torch_aten/generate_prims_signatures.py
Comment (justinchuby) [reply onnxscript/function_libs/tools/torch_aten/generate_prims_signatures.py:1]: Looks like I forgot to rename. I can do that later.

**Pull request merged**